packages/npm.rb

```diff
@@ -0,0 +1,51 @@
+# Adapted from Arch Linux npm PKGBUILD at:
+# https://github.com/archlinux/svntogit-community/raw/packages/npm/trunk/PKGBUILD
+
+require 'package'
+
+class Npm < Package
```

Comment:
npm comes with node which is installable with nodebrew.  No need for a package.

Reply:
Removed!